### ISSUE TITLE
Console error when using mdcList directive with nonInteractive property

### ISSUE THREAD

**Issue by AXeL-dev** [src-zone/material#690]
Hello,

I just wanted to report the following console error when using `mdcList` directive with `nonInteractive` property:

![image](https://user-images.githubusercontent.com/30635693/86538895-d862bf00-bef0-11ea-89fe-296b450bb6d4.png)

My source code was taken from https://blox.src.zone/material/components/list, below a short part of it:
```
<ul mdcList [nonInteractive]="true">
  <li mdcListItem>
    <i mdcListItemGraphic class="material-icons">share</i>
    <span mdcListItemText>
      Proxy
      <span mdcListItemSecondaryText>Used to bypass CORS limitations on browsers</span>
    </span>
    <div mdcListItemMeta mdcSelect>
      <select mdcSelectControl [(ngModel)]="settings.proxy.url">
        <option *ngFor="let proxy of getProxies()" [value]="proxy.url">{{ proxy.name }}</option>
      </select>
      <label mdcFloatingLabel></label>
    </div>
  </li>
  ...
</ul>
```
The non interactive effect seems to work but i always keep getting the console error (when used inside a `*ngFor` loop, it keeps printing a bunch of console errors).

Also, down below my dependencies versions:

![image](https://user-images.githubusercontent.com/30635693/86539063-c7ff1400-bef1-11ea-80ee-ac4a1664207c.png)

Thanks.

**Comment by jorgevargas1989** [comment]
I will have a look into this next friday.

**Comment by jorgevargas1989** [comment]
Sorry, friday was a bit busy. I just committed the likely fix, so you could check the master branch now. Or wait for the next release, but that will probably be the end of this week.

**Comment by jorgevargas1989** [comment]
Should be fixed in the master branch. If you could confirm the fix, that would be nice. Otherwise we will publish a patch release from that branch later this week.

**Comment by AXeL-dev** [comment]
I've already patched my node_modules as a temporary workaround, so yeah it's working:

![image](https://user-images.githubusercontent.com/30635693/87332216-2c922280-c533-11ea-9330-37da26b28ff9.png)

But i'll wait for the patch of this week so i can get a definitive solution. Thanks!

**Comment by jorgevargas1989** [comment]
Released in 0.18.1
